packages/server-wallet/src/config/defaults.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-process-env */
+import {
+  LoggingConfiguration,
+  NetworkConfiguration,
+  OptionalDatabaseConfiguration,
+  OptionalServerWalletConfig,
+  ServerWalletConfig,
+} from './types';
+
+export const defaultDatabaseConfiguration: OptionalDatabaseConfiguration & {
+  connection: {host: string; port: number};
+} = {
+  pool: undefined,
+  debug: false,
+  connection: {host: 'localhost', port: 5432, user: 'postgres'},
+};
+
+/**
+ * Network config
+ */
+
+export const defaultNetworkConfiguration: NetworkConfiguration = {
+  chainNetworkID: '0x00',
+  erc20Address: undefined,
+  ethAssetHolderAddress: undefined,
+  erc20AssetHolderAddress: undefined,
```

Comment:
I don't think `erc20AssetHolderAddress` nor `erc20Address` should be here, and `ethAssetHolderAddress` is always `0x0000000000000000000000000000000000000000`

Reply:
Why don't you think they should be there?

Reply:
There is no such thing as an "ERC20 asset" — it is an asset class. The assets are things like DAI, GRT, etc.

Reply:
> `ethAssetHolderAddress` is always `0x0000000000000000000000000000000000000000`

Why would it always be the zero address?